src/PrestaShopBundle/Command/UpdateSchemaCommand.php

```diff
@@ -58,16 +58,12 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $config = include __DIR__ . '/../../../app/config/parameters.php';
-        if ($input->getOption('env') === 'test') {
-            $this->dbName = 'test_' . $config['parameters']['database_name'];
-        } else {
-            $this->dbName = $config['parameters']['database_name'];
-        }
+        $container = $this->getContainer();
 
-        $this->dbPrefix = $config['parameters']['database_prefix'];
+        $this->dbName = $container->getParameter('database_name');
+        $this->dbPrefix = $container->getParameter('database_prefix');
 
-        $this->em = $this->getContainer()->get('doctrine')->getManager();
+        $this->em = $container->get('doctrine')->getManager();
```

Comment:
You can use DI the EntityManager

Reply:
What do you mean?

Reply:
EntityManager is a dependency. You can inject it in the constructor ?

Reply:
Not as I know because of this: https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Service/Database/Upgrade.php#L39 we don't have the container here :/
And the command already play with `getContainer`

Reply:
:ok_hand:  for me :)